### PR TITLE
Demo page to showcase adding playback control UI during Ad playback with IMA SDK

### PR DESCRIPTION
**Pull request by luckygoyal-bitmovin**
The demo page showcases adding playback controls UI to Bitmovin player during Ads playback with IMA SDK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ This showcases are build around the Bitmovin Adaptive Streaming Player, demonstr
    * [separatedSubtitleAudioSettings](playerUi/separatedAudioSubtitleSettings.html): Extracted audio tracks and subtitle settings from the settings panel direct into the controlBar
    * [labeling](playerUi/labeling.html): Change Languages, Quality and Subtitles labels
    * [customForwardRewindButtons](playerUi/customForwardRewindButtons.html): Custom Forward & Rewind Buttons
+   * [playbackControlsUIWithImaAds](playerUi/playbackControlsUIWithImaAds.html) : Adding playback controls UI during Ads playback with IMA SDK
```
Comment (HalfbyteHeroes) [inline README.md:41]: Should be `playbackControlsUiWithImaAds` for consistency

File: playerUi/playbackControlsUIWithImaAds.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    key: "YOUR-PLAYER-LICENSE-KEY",
+    advertising: {
+        adBreaks: [
+        {
+            tag: {
+              url:
+              'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpostpod&cmsid=496&vid=short_onecue&correlator=',
+              type: 'vmap'
+            },
+        }
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:140]: Wrong indentation

File: playerUi/playbackControlsUIWithImaAds.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        <p>For more information about the bitmovin player, please have a look at our online <a
+          href="//bitmovin.com/docs" target="_blank">Documentation</a>.</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  var conf = {
+    key: "YOUR-PLAYER-LICENSE-KEY",
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:131]: Inconsistent usage of double and single quotes - I'd suggest using single quotes

File: playerUi/playbackControlsUIWithImaAds.html [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+              url:
+              'https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpostpod&cmsid=496&vid=short_onecue&correlator=',
+              type: 'vmap'
+            },
+        }
+        ],
+    },
+    playback: {
+      autoplay: true,
+      muted:true
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:145]: Missing whitespace

File: playerUi/playbackControlsUIWithImaAds.html [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+      return context.isAd;
+    },
+  }], {metadata: {}});
+
+  var sourceConfig = {
+    title: "Bitmovin Player",
+    description: "This demo showcases the possibility to add playback controls UI during Ad playback!",
+    dash: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/mpds/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.mpd",
+    hls: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/m3u8s/f08e80da-bf1d-4e3d-8899-f0f6155f6efa.m3u8",
+    progressive: "https://bitmovin-a.akamaihd.net/content/MI201109210084_1/MI201109210084_mpeg-4_hd_high_1080p25_10mbits.mp4",
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:188]: Inconsistent usage of double and single quotes - I'd suggest using single quotes

File: playerUi/playbackControlsUIWithImaAds.html [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+      autoplay: true,
+      muted:true
+    },
+  };
+
+  var player = new bitmovin.player.Player(document.getElementById('player'), conf);
+
+  // UI for IMA SDK
+  // Create a separate UI manager that shows a custom UI for all ads
+  var imaUIManager= new bitmovin.playerui.UIManager(player, [{
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:153]: Missing whitespace

File: playerUi/playbackControlsUIWithImaAds.html [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+    ui: new bitmovin.playerui.UIContainer({
+      userInteractionEventSource: document.getElementById('player'), // show controls on user interaction
+      //hideDelay: -1, // always show UI
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:156]: Please remove this commented-out line

File: playerUi/playbackControlsUIWithImaAds.html [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+      hideDelay: 3000, // hide controls in 3 second
+      components: [
+        new bitmovin.playerui.Container({
+        }),
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:160]: Superfluous line break

File: playerUi/playbackControlsUIWithImaAds.html [hunk 9]
```diff
@@ -0,0 +1,199 @@
...
+                playbackButton = new bitmovin.playerui.PlaybackToggleButton(),
+                new bitmovin.playerui.VolumeToggleButton(),
+                new bitmovin.playerui.VolumeSlider(),
+              ],
+              cssClasses: ['controlbar-bottom'],
+            }),
+          ],
+        cssClasses: ['adControlBar']
+      }),
+      ],
```
Comment (HalfbyteHeroes) [inline playerUi/playbackControlsUIWithImaAds.html:174]: Wrong indentation I think

**Review by HalfbyteHeroes — APPROVED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (luckygoyal-bitmovin) [reply README.md]: done

File: playerUi/playbackControlsUIWithImaAds.html (hunk 2 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

File: playerUi/playbackControlsUIWithImaAds.html (hunk 3 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

File: playerUi/playbackControlsUIWithImaAds.html (hunk 4 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

**Review by hanneslinder — APPROVED** [review]

File: playerUi/playbackControlsUIWithImaAds.html (hunk 5 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated as suggested

File: playerUi/playbackControlsUIWithImaAds.html (hunk 6 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

File: playerUi/playbackControlsUIWithImaAds.html (hunk 7 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: removed

File: playerUi/playbackControlsUIWithImaAds.html (hunk 8 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

File: playerUi/playbackControlsUIWithImaAds.html (hunk 9 above)
Comment (luckygoyal-bitmovin) [reply playerUi/playbackControlsUIWithImaAds.html]: updated

**Pull request merged**